tsg/ranker/base.py

```diff
@@ -96,9 +95,10 @@ def or_score_calc(query_terms, index_dictionary_path=DICTIONARY_PATH,
     query_length = {} # Holds score^2 for Length normalization at end
     doc_length = {}
     for term in query_terms:
-        query_term_weight = calculate_query_term_weight(term,query_terms,
-            index_dictionary_path, index_info_path)
         term_documents = get_dictionary_term_list(term, index_dictionary_path)
```

Comment:
Hmmmm. I'll double check and merge :)

Brian A. Ignacio <notifications@github.com> schrieb am Di., 10. Mai 2016,
20:29:

> In tsg/ranker/base.py
> <https://github.com/moritzschaefer/the-search-engine/pull/44#discussion_r62654901>
> :
>
> > @@ -96,9 +95,10 @@ def or_score_calc(query_terms, index_dictionary_path=DICTIONARY_PATH,
> >      query_length = {} # Holds score^2 for Length normalization at end
> >      doc_length = {}
> >      for term in query_terms:
> > -        query_term_weight = calculate_query_term_weight(term,query_terms,
> > -            index_dictionary_path, index_info_path)
> >          term_documents = get_dictionary_term_list(term, index_dictionary_path)
>
> I don-t understand why you see this code. If you check the file doc_freq
> is also there.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/moritzschaefer/the-search-engine/pull/44/files/cbe6bf02a81f8a6c94806d515f983efe6f8ceb05#r62654901>
>